### ISSUE TITLE
Activation functions as static context instead of constant

### ISSUE THREAD

**Issue by xxMrPHDxx** [CodingTrain/Toy-Neural-Network-JS#78]
We are currently using constant to declare those activation functions. 
I think it will be much better if we use static context instead to declare them.

[Here](https://gist.github.com/xxMrPHDxx/f05b1ae9c722e4cbd99fd41f722fb5dc) I included the gist of mine as a reference.

**Comment by AR-234** [comment]
@xxMrPHDxx ive done something similar in #71 (only done on the current build on purpose), what I've done different is only make the functions to getters, which is in my opinion a little bit cleaner when you write the code where the functions are inserted, but I guess its a personal preference.

**Comment by Adil-Iqbal** [comment]
I agree with the idea that the activation functions should be moved into their own file.  There are a ton of activation functions that have yet to be implemented because of the current limitations of the ```ActivationFunction``` class.  The class itself will probably go through several changes in the future, especially if we are implementing activation functions with alpha and lambda values. Not to mention if we implement activations that go past a single-fold x, like the ```softmax``` and ```maxout``` activations.
